pages/about.js

```diff
@@ -0,0 +1,45 @@
+import Dropdown from "../components/Dropdown"
+import Link from 'next/link'
+import { useRouter } from "next/router"
+import { useState } from 'react'
+import React, {Component} from "react"
+import { getRouteMatcher } from "next/dist/shared/lib/router/utils/route-matcher"
+import { View, StyleSheet } from 'react'
+
+export default function about(){
+    const router = useRouter()
+    return (
+        <div>
+            {/* begin nav bar */}
+            <nav className="bg-[#292828] border-gray-200 border-b-2 px-2 sm:px-4 py-2.5 shadow-md">
+                <div className="container text-white flex flex-wrap justify-between items-center mx-auto">
+                <Link href={{pathname: "/", query: {userID: router.query.userID}}}>
```

Comment:
Make sure for the about us that since we want others to be able to access without an account, we will need to conditionally replace the link depending on if a userID was passed as query params in the URL